docs/reference/reference-architectures/elastic-cloud-architecture.asciidoc

```diff
@@ -0,0 +1,171 @@
+[[elastic-cloud-architecture]]
+== Elastic Cloud Hot-Frozen Architecture for Time Series Data
+++++
+<titleabbrev>Architecture: Elastic Cloud hot-frozen</titleabbrev>
+++++
+
+The Hot-Frozen Elasticsearch cluster architecture is cost optimized for large time-series datasets while keeping all of the data **fully searchable**. There is no need to "re-hydrate" archived data. In this architecture, the hot tier is primarily used for indexing and immediate searching (1-3 days) with a majority of the search being handled by the frozen tier. Since the data is moved to searchable snapshots in an object store, the cost of keeping all of the data searchable is dramatically reduced.
+
+
+TIP: This architecture includes all the necessary components of the Elastic Stack and is not intended for sizing workloads, but rather as a basis to ensure the architecture you deploy is built with sound recommendations. 
```

Comment:
```suggestion
TIP: This architecture includes all the essential components of the Elastic Stack. It's designed to ensure stability and flexibility, based on expert recommendations, but is not intended for sizing workloads.
```

 I meant to comment about this in the preview but I think it got lost.
 
 Applies to all of these TIPS

Reply:
I missed this and added it locally. This is addressed in a newer commit.